Source/JavaScriptCore/bytecode/AccessCase.cpp

```diff
@@ -1789,7 +1789,7 @@ void AccessCase::generateImpl(AccessGenerationState& state)
     StructureStubInfo& stubInfo = *state.stubInfo;
     JSValueRegs valueRegs = state.valueRegs;
     GPRReg baseGPR = state.baseGPR;
-    GPRReg thisGPR = stubInfo.thisValueIsInThisGPR() ? state.u.thisGPR : baseGPR;
+    GPRReg thisGPR = stubInfo.thisValueIsInExtraGPR() ? state.extraGPR : baseGPR;
```

Comment:
This is necessary to simplify / clean up StructureStubInfo initialization from DFG::UnlinkedStructureStubInfo. To avoid undefined behavior of C, we need to use the right field of the union for each type, but this makes code ugly.
By not using union, we can make initialization significantly simpler.
For now, I'll define getters `propertyGPR()` etc. and use it instead.